src/main/java/uk/gov/hmcts/darts/arm/api/impl/ArmDataManagementApiImpl.java

```diff
@@ -43,6 +47,11 @@ public void deleteResponseBlob(String blobName) {
         armService.deleteResponseBlob(getArmContainerName(), blobName);
     }
 
+    @Override
+    public UpdateMetadataResponse updateMetadata(String externalRecordId, OffsetDateTime eventTimestamp) {
+        return armApiService.updateMetadata(externalRecordId, eventTimestamp);
```

Comment:
I'm not quite sure what this part of the Jira ticket means:

> Input parameter will be RecordId from External Object Directory table and calculated event_date. For now pass the event date as 7 years from current Date. There will be a separate ticket ([DMP-2057](https://tools.hmcts.net/jira/browse/DMP-2057)) to calculate event_date as it depends upon Retention Service.

Does this mean we should be ignoring the `eventTypestamp` param here temporarily, and evaulating the second arg to `updateMetadata` as current time + 7 years, or are we ok to just accept the value from the caller?

Reply:
I was hoping it would be ok to accept the date/time from the caller as we have here - I was confused myself and thought I needed to plug it into maybe `ArmResponseFilesProcessorImpl` - but think @hemantasharma1129 said that it will be included in another area of retention that is already ticketed. So will leave it for Hemanta to call out here.